### DISCUSSION TITLE
Superb job!

### DISCUSSION THREAD

**Discussion by billybobbain** [uxbox/uxbox#1747] [Show and tell]
I got the docker version going in minutes and now have google auth, smtp relay done and it's working great. I hope you don't take this the wrong way. This is a package for professionals, but I set this up for my 9 year old artist daughter to collaborate with her friends.  It's a hit already. I am a dev & will look for things to help out with. Great job!

**Comment by hydrocat** [comment]
I agree, it's a great project !
